### DISCUSSION TITLE
Enabling GitHub Discussions for Quest

### DISCUSSION THREAD

**Discussion by alexwarren** [textadventures/quest#1244] [Announcements]
Just a quick announcement to say I've enabled GitHub Discussions here. You can see more details in the [parent announcement](https://github.com/orgs/textadventures/discussions/4).

**Comment by KVonGit** [comment]
Hurray!

I started using Discussions in my repos a few months ago, and it's pretty sweet. You can convert Discussions to Issues and vice-versa. There are a few file formats we can't simply attach to comments, but we can easily just zip the file(s) and attach the ZIP.

Being able to pin discussions is also very nice, and they actually appear at the top of the page as expected (unlike Discord).
